pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -101,26 +103,20 @@ func (configgen *ConfigGeneratorImpl) BuildDeltaClusters(proxy *model.Proxy, upd
 	services := make([]*model.Service, 0)
 	// In delta, we only care about the services that have changed.
```

Comment:
legacy?

Reply:
what do you mean?

Reply:
I mean both service and DestinarionRule cares